deployments/polygon.json

```diff
@@ -52,12 +52,13 @@
   "ReceiverStargateV2": "0x1493e7B8d4DfADe0a178dAD9335470337A3a219A",
   "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
   "EmergencyPauseFacet": "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61",
-  "AcrossFacetV3": "0x5052fc5c7486162deDf7458E1f7c6ABaFbcd6895",
+  "AcrossFacetV3": "0x6e00e0a7685ca22c288d56d9e7924746b5043ee7",
```

Comment:
On some mainnet diamonds we have several facets of the same type registered and this is one example. Your script does not recognize this  and (erroneously) uses the first version (but the latest one is the last version). 
We need to clean this up.

Could you maybe create a list of facets that are outdated and need to be removed from our PROD diamonds?